src/actions/index.js

```diff
@@ -30,6 +35,15 @@ export const deleteTodo = (id) => {
     id,
   };
 };
+
+export const completeTodo = (id, date) => {
+  return {
+    type: COMPLETE_TODO,
+    id,
+    isTaskComplete: true,
+    completeTime: getDays(date.getTime(), new Date().getTime()),
```

Comment:
// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now

```suggestion
    completeTime: getDays(date.getTime(), Date.now()),
```